docs/CLI.md

```diff
@@ -202,6 +202,10 @@ Make calling deprecated APIs throw helpful error messages. Useful for easing the
 
 Alias: `-e`. Use this flag to show full diffs and errors instead of a patch.
 
+### `--fakeTimers=<json string>`
```

Comment:
not sure if it makes sense to pass these sorts of options from the CLI. seems more natural as a config option

Reply:
To pass an object like this is not fun, of course. If I get it right, Yargs [should allow](https://github.com/yargs/yargs/blob/main/docs/tricks.md#objects) `jest --fakeTimers.enableGlobaly=true` or simply `jest --fakeTimers.enableGlobaly`. Verbose, but might be useful. Have to check if and how it works.